templates/base.html.twig

```diff
@@ -9,11 +9,12 @@
 		{{ msg('app-title') }}
 	</title>
 	{% block stylesheets %}
-		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
+		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />		
 	{% endblock %}
 	{% block scripts %}
 		<script type="text/javascript" src="//tools-static.wmflabs.org/cdnjs/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
 		<script type="text/javascript" src="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/js/bootstrap.min.js"></script>
+		<script src="https://cdn.jsdelivr.net/npm/sortablejs@1.14.0/Sortable.min.js"></script>
```

Comment:
This (which is https://github.com/SortableJS/Sortable ) doesn't seem to be a table-sorting library, but rather is for sorting *lists*. The one I was suggesting is https://github.com/HubSpot/sortable

Libraries should also be loaded from the tools-static.wmflabs.org/cdnjs service.

Reply:
Actually it was working for me. But will surely look into this

Reply:
Oh, sorry perhaps I'm testing it wrongly.

I was expecting to see sorting icons on the table headers, is that right? I don't see anything like that, and there are no errors in the JS console.